dnf/cli/main.py

```diff
@@ -158,6 +158,8 @@ def resolving(cli, base):
     # Run the transaction
     try:
         base.do_transaction()
+        base.plugins.run_transaction()
+        logger.info(_('Complete!'))
```

Comment:
Please rather move these lines into an ```else``` clause of the ```try``` statement. The rationale: ```Additionally, for all try/except clauses, limit the try clause to the absolute minimum amount of code necessary. Again, this avoids masking bugs.``` [https://www.python.org/dev/peps/pep-0008/]